typing/parmatch.ml

```diff
@@ -1979,8 +1991,13 @@ let irrefutable pat = le_pat pat omega
    trivial computations (tag/equality tests).
    Patterns containing (lazy _) subpatterns are active. *)
 
+(* I'm not sure this is correct for [Tpat_exception], I suspect it isn't.
+   However [inactive] is called only from [fluid] which is called only by
+   [Translcore.transl_function] where [Tpat_exception] doesn't appear, so it's
+   not critical.
+   But I'd be happy if someone could make a decision. *)
```

Comment:
If you're saying that `Tpat_exception` should not occur here and its not clear what this function should do if it still appeared, you should probably `assert false`, no?